.vscode/launch.json

```
@@ -18,13 +18,13 @@
             "cargo": {
                 "args": [
                     "build",
-                    "--bin=nebula_astralis", // ToDo: Change bin/package titles
-                    "--package=nebula_astralis", // ToDo: Change bin/package titles
+                    "--bin=nebula_astralis",
+                    "--package=azuna.nebula_astralis",
```

Package name is in the Cargo.toml

```suggestion
                    "--package=nebula_astralis",
```